<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OChReDialog</class>
 <widget class="QDialog" name="OChReDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>评价</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>报修ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtId">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxKind">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>水</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>家具</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>墙面/地面</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>燃气</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edtProblem">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>评价</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交评价</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChange">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>改变描述</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>当前类型</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtKind">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="edtEvaluate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>星级</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxStar">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnChange</sender>
   <signal>clicked()</signal>
   <receiver>OChReDialog</receiver>
   <slot>change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSubmit</sender>
   <signal>clicked()</signal>
   <receiver>OChReDialog</receiver>
   <slot>submit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>change()</slot>
  <slot>submit()</slot>
 </slots>
</ui>
